src/components/common/Search/Search.tsx

```diff
@@ -256,26 +275,37 @@ export class Search extends React.Component<SearchProps, SearchState> {
 
   render() {
     const { assetId, query, isModalOpen, advancedSearchQuery } = this.state;
-    const { assets, advancedSearch, rootAssetSelect, strings } = this.props;
+    const {
+      assets,
+      advancedSearch,
+      rootAssetSelect,
+      strings,
+      styles,
+    } = this.props;
 
     this.lang = { ...defaultStrings, ...strings };
 
     const { changeSearch, clear, searchPlaceholder, search } = this.lang;
-
     return (
       <React.Fragment>
         <InputGroup compact={true}>
           {rootAssetSelect && (
-            <RootAssetSelectStyled
+            <RootAssetSelect
               onAssetSelected={this.onAssetSelected}
               assets={assets}
               assetId={assetId}
+              styles={{
+                select: { width: '35%', ...(styles && styles.rootAssetSelect) },
```

Comment:
Can we move ` width: '35%'` to the styled-components definition of `RootAssetSelect ` and get rid of destructuring here?

Reply:
`RootAssetSelect` has no styled-component definition, just a `Select` from `antd`. You think it would be better to add one?

Reply:
This better?